components/atomic/Loader/index.js

```diff
@@ -0,0 +1,9 @@
+import * as Styled from "./styles";
+
+function Loader(props) {
+  return <Styled.Loader {...props} />;
+}
+
+Loader.displayName = "Atomic.Loader";
+
+export default Loader;
```

Comment:
Lil Styled version of the spinner from the UI lib.  Styles just center it horizontally and vertically in its parent.